packages/jellyfish-api-core/__tests__/category/account/futureswap.test.ts

```diff
@@ -0,0 +1,922 @@
+  await testing.token.create({ symbol: 'BTC', collateralAddress })
+  await testing.generate(1)
+  await testing.token.mint({ symbol: 'BTC', amount: 20000 })
+  await testing.generate(1)
+
+  // loan scheme
+  await testing.container.call('createloanscheme', [100, 1, 'default'])
+  await testing.generate(1)
+
+  // price oracle
+  const priceFeeds = [
+    { token: 'DFI', currency: 'USD' },
+    { token: 'BTC', currency: 'USD' },
+    { token: 'TSLA', currency: 'USD' },
+    { token: 'DUSD', currency: 'USD' }
+  ]
+
+  const addr = await testing.generateAddress()
+  oracleId = await testing.rpc.oracle.appointOracle(addr, priceFeeds, { weightage: 1 })
+  await testing.generate(1)
+
+  const timestamp = Math.floor(new Date().getTime() / 1000)
+  await testing.rpc.oracle.setOracleData(
+    oracleId,
+    timestamp,
+    {
+      prices: [
+        { tokenAmount: '1@DFI', currency: 'USD' },
+        { tokenAmount: '10000@BTC', currency: 'USD' },
+        { tokenAmount: '2@TSLA', currency: 'USD' },
+        { tokenAmount: '1@DUSD', currency: 'USD' }
+      ]
+    }
+  )
+  await testing.generate(1)
+
+  // collateral tokens
+  await testing.rpc.loan.setCollateralToken({
+    token: 'DFI',
+    factor: new BigNumber(1),
+    fixedIntervalPriceId: 'DFI/USD'
+  })
+  await testing.generate(1)
+
+  await testing.rpc.loan.setCollateralToken({
+    token: 'BTC',
+    factor: new BigNumber(0.5),
+    fixedIntervalPriceId: 'BTC/USD'
+  })
+  await testing.generate(1)
+
+  // loan token
+  await testing.rpc.loan.setLoanToken({
+    symbol: 'TSLA',
+    fixedIntervalPriceId: 'TSLA/USD'
+  })
+  await testing.generate(1)
+
+  await testing.rpc.loan.setLoanToken({
+    symbol: 'DUSD',
+    fixedIntervalPriceId: 'DUSD/USD'
+  })
+  await testing.generate(1)
+
+  idTSLA = await testing.token.getTokenId('TSLA')
+  idDUSD = await testing.token.getTokenId('DUSD')
+
+  // create a vault and take loans
+  const vaultAddr = await testing.generateAddress()
+  const vaultId = await testing.rpc.loan.createVault({
+    ownerAddress: vaultAddr,
+    loanSchemeId: 'default'
+  })
+  await testing.generate(1)
+
+  await testing.rpc.loan.depositToVault({
+    vaultId: vaultId, from: collateralAddress, amount: '100000@DFI'
+  })
+
+  // wait till the price valid.
+  await testing.container.waitForPriceValid('TSLA/USD')
+
+  // take multiple loans
+  await testing.rpc.loan.takeLoan({
+    vaultId: vaultId,
+    to: collateralAddress,
+    amounts: ['300@TSLA', '500@DUSD']
+  })
+  await testing.generate(1)
+
+  // Futures setup
+  // set the dfip2203/active to false
+  await testing.rpc.masternode.setGov({ [attributeKey]: { 'v0/params/dfip2203/active': 'false' } })
+  await testing.generate(1)
+
+  // set dfip2203 params
+  await testing.rpc.masternode.setGov({ [attributeKey]: { 'v0/params/dfip2203/reward_pct': `${futRewardPercentage}`, 'v0/params/dfip2203/block_period': `${futInterval}` } })
+  await testing.generate(1)
+
+  // activat the dfip2203/active now
```

Comment:
```suggestion
  // set the dfip2203/active to true
```
Like how you wrote in line 112